fe1-web/src/features/events/components/EventList.tsx

```diff
@@ -43,16 +43,47 @@ const EventList = () => {
     throw new Error('Cannot show an event list if you are not connected to a lao!');
   }
 
+  const eventListSelector = useMemo(() => makeEventListSelector(laoId.valueOf()), [laoId]);
+  const events = useSelector(eventListSelector);
+
+  const [{ pastEvents, currentEvents, futureEvents }, setEvents] = useState<{
+    pastEvents: EventState[];
+    currentEvents: EventState[];
+    futureEvents: EventState[];
+  }>(() => {
+    const [newPastEvents, newCurrentEvents, newFutureEvents] = categorizeEventsByTime(
+      Timestamp.EpochNow().valueOf(),
```

Comment:
it seems a bit odd in the API that you'd pass a number, instead of a `Timestamp`. 
You might want to do the `valueOf` conversion internally.